api/models/ml_model.py

```diff
@@ -4,6 +4,18 @@
 from .base import SnakeModel
 
 
+MODELS_DEVICES = {
+    "Jetson": ["ssd_mobilenet_v2_coco", "ssd_mobilenet_v2_pedestrian_softbio", "openpifpaf_tensorrt"],
+    "EdgeTPU": ["mobilenet_ssd_v2", "pedestrian_ssd_mobilenet_v2", "pedestrian_ssdlite_mobilenet_v2", "posenet"],
+    "Dummy": ["openvino", "openpifpaf_tensorrt", "mobilenet_ssd_v2", "openpifpaf", "yolov3", "ssd_mobilenet_v2_coco",
+              "ssd_mobilenet_v2_pedestrian_softbio", "posenet", "pedestrian_ssd_mobilenet_v2",
+              "pedestrian_ssdlite_mobilenet_v2"],  # All available models.
+    "x86": ["mobilenet_ssd_v2", "openvino", "openpifpaf", "openpifpaf_tensorrt", "yolov3"],
+    "x86-gpu": ["mobilenet_ssd_v2", "openvino", "openpifpaf", "openpifpaf_tensorrt", "yolov3"],
+
```

Comment:
why the extra line?

Reply:
It is not extra, it is another device "x86" and  "x86-gpu".

Reply:
nono I meant the empty line before the }

Reply:
Oh, sorry. Done!